app/helpers/messages_helper.rb

```diff
@@ -0,0 +1,12 @@
+###
+#  Helper module for broadcast message related markup
+###
+module MessagesHelper
+  def link_to_edit_message (message)
+    link_to "Edit message", [:edit, message], class: 'btn btn-sm btn-default edit-message glyphicon glyphicon-pencil'
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.